src/lib/components/TxFilterSelection.tsx

```diff
@@ -80,13 +86,17 @@ export const TxFilterSelection = forwardRef<
     const wasm = useWasmConfig({ shouldRedirect: false });
     const move = useMoveConfig({ shouldRedirect: false });
 
+    const { chain } = useCurrentChain();
+    const isInitia = chain.bech32_prefix === "init";
```

Comment:
Please create a custom hook named `useInitia`. Rather than comparing with `bech32_prefix`, let's use the `chain` found in the chain's configuration. You can access it using:
```ts
const { chainConfig: {chain}} = useCelatoneApp();
```

Reply:
added.